controllers/activemqartemis_controller_test.go

```diff
@@ -9598,14 +9598,62 @@ var _ = Describe("artemis controller", func() {
 			}
 		})
 
-		It("secure connections with multiple wildcard DNS names", func() {
+		It("secure connections with openshift serving certificate", func() {
+			if os.Getenv("USE_EXISTING_CLUSTER") == "true" && isOpenshift {
+				crd := generateArtemisSpec(defaultNamespace)
+
+				tlsSecretName := crd.Name + "-cert-provided-secret"
```

Comment:
nice, provided seems the perfect name and naming convention over config is a wise approach

Reply:
I had to reduce the length of the suffix to avoid issues for names longer than 63 characters. The new suffix is `cp-secret`